modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Controller/PublisherMethodsController.cs

```diff
@@ -64,10 +64,10 @@ public class PublisherMethodsController : IMethodController {
         /// <summary>
         /// Handler for GetConfiguredEndpoints DM
         /// </summary>
-        public async Task<List<PublishedNodesResponseApiModel>> GetConfiguredEndpointsAsync() {
+        public async Task<List<PublishNodesRequestApiModel>> GetConfiguredEndpointsAsync() {
```

Comment:
@monikavar, please add unit and integration tests for `GetConfiguredEndpoints_V1` direct method.

Reply:
missed to push the file in earlier commit. done now.